nibabel/testing/__init__.py

```diff
@@ -164,3 +166,9 @@ def __init__(self, *args, **kwargs):
         warnings.warn('catch_warn_reset is deprecated and will be removed in '
                       'nibabel v3.0; use nibabel.testing.clear_and_catch_warnings.',
                       FutureWarning)
+
+
+def skipif_no_slow_tests(func):
```

Comment:
Right. There's no DRY concern here; there's no repetition. I will just do it now, but this adds code that's (very slightly) more complex than currently needed... which I've been well-trained to avoid :)